lib/players/computer.rb

```diff
@@ -0,0 +1,84 @@
+class Player::Computer < Player
+
+	 COMBINATIONS = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]]
+
+	def move(board)
```

Comment:
This method is very long. Consider breaking it up into smaller helper methods (your comments delineate possible helper methods well).